core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -107,34 +107,24 @@
       .withPartitionPath("data_bucket=2") // easy way to set partition data for now
       .withRecordCount(1)
       .build();
-  static final DataFile FILE_D = DataFiles.builder(SPEC)
-      .withPath("/path/to/data-d.parquet")
-      .withFileSizeInBytes(10)
-      .withPartitionPath("data_bucket=3") // easy way to set partition data for now
-      .withRecordCount(1)
-      .build();
-  static final DataFile FILE_PARTITION_0 = DataFiles.builder(SPEC)
-      .withPath("/path/to/data-0.parquet")
-      .withFileSizeInBytes(10)
-      .withPartition(TestHelpers.Row.of(0))
-      .withRecordCount(1)
-      .build();
-  static final DataFile FILE_PARTITION_1 = DataFiles.builder(SPEC)
```

Comment:
This is just some minor cleanup to unify some of the test tables used in "TestMetadataTableScans", I had added these sometime ago and noticed they are actually the same as the more commonly used FILES_A, FILES_B, etc.